src/main/java/com/google/opensesame/projects/ProjectData.java

```diff
@@ -26,6 +28,8 @@
   private Integer numMentors = null;
   private String repositoryId = null;
   private List<UserData> mentors = null;
+  private String readmeRawUrl = null;
+  private String readmeHtmlUrl = null;
```

Comment:
nit: these names are kind of confusing. It sounds a lot like Raw is describing the URL and not the readme. These seems like a difficult set of fields to name so it's not a big deal. Maybe we could stick to the API's convention of ```Downloadable vs HTML```